### PR TITLE
SISRP-32543 - Surfaces term honors on Academic Summary

### PR DESCRIPTION
**Pull request by lyttam**
https://jira.berkeley.edu/browse/SISRP-32543

This PR satisfies the following requirement, does a little CSS refactoring, and removes some unrelated debug/log statements that appear to have been accidentally left in.

- Academic Summary page:  on each term, surface "Deans List" and/or "Honors to Date" awards if they were awarded during that term

### CODE REVIEW

**Comment by lyttam** [comment]
Failing Travis because it doesn't recognize the Ruby 2 syntax I tried to sneak past it. Investigating.

File: app/models/my_academics/college_and_level.rb [hunk 1]
```diff
@@ -53,6 +57,37 @@ def parse_hub_holds(response)
       holds
     end
 
+    def parse_hub_award_honors(response)
+      honors = sort_award_honors response.dig(:feed, 'student', 'awardHonors')
+      honors_by_term = {}
+      honors&.each do |honor|
+        term_id = honor.dig('term', 'id')
```
Comment (redconfetti) [inline app/models/my_academics/college_and_level.rb]: @lyttam - This is what Travis is barking about
```
SyntaxError: /home/travis/build/ets-berkeley-edu/calcentral/app/models/my_academics/college_and_level.rb:63: syntax error, unexpected tDOT
      honors&.each do |honor|
```

Comment (redconfetti) [inline app/models/my_academics/college_and_level.rb]: Anne clarified that what I perceived as a typo is [actually a valid Ruby operator](http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/).

Comment (redconfetti) [inline app/models/my_academics/college_and_level.rb]: @lyttam - Perhaps we need to update the Ruby version to 9.1.5.0 in [travis.yml](https://github.com/ets-berkeley-edu/calcentral/blob/2b36640d/.travis.yml), like we did in [.rvmrc](https://github.com/ets-berkeley-edu/calcentral/blob/2b36640d/.rvmrc#L15)

**Comment by raydavis** [comment]
@lyttam Looks like your PR keeps getting the same mysterious Travis failure as mine -- let me know if you spot anything!

**Comment by lyttam** [comment]
@raydavis I've been banging away at it but haven't figured out the cause.  JSHint error?  But JSHint passes when I run it in my local environment.

**Comment by raydavis** [comment]
Same here, @lyttam . Locally, I get:

```
calcentral> JRUBY_OPTS="--dev -J-Xmx900m" DISPLAY=:99.0 LOGGER_LEVEL=WARN TRAVIS_NODE_VERSION="4" bundle exec rake travis
...
Running JSHint - detect errors and potential problems in JavaScript code
Running jscs - JavaScript Style Checker
# successful exit
```

And the same for @redconfetti 's most recent PR on Travis-CI: 

```
Running JSHint - detect errors and potential problems in JavaScript code
Running jscs - JavaScript Style Checker

travis_time:end:1f9cba5c:start=1501110516945384904,finish=1501110794049974044,duration=277104589140
[0K
[32;1mThe command "bundle exec rake" exited with 0.[0m
```

But my PR's Travis log just says:

```
Running JSHint - detect errors and potential problems in JavaScript code
rake aborted!
JSHint failed!
/home/travis/build/ets-berkeley-edu/calcentral/lib/tasks/travis.rake:5:in `block in (root)'
Tasks: TOP => default => travis
(See full trace by running task with --trace)

travis_time:end:1300efde:start=1501185994119912194,finish=1501186490633813018,duration=496513900824
[0K
[31;1mThe command "bundle exec rake" exited with 1.[0m
```

**Comment by lyttam** [comment]
@raydavis I'm running rake locally and getting the same result as you.

I thought it might be the case that the nonzero return code is being set before jshint runs, but I don't see evidence of that in the logs.

**Comment by lyttam** [comment]
This is ready to merge now.  As a temporary workaround to the Travis issue, I set the Ubuntu version explicitly in travis.yml.

File: .travis.yml [hunk 2]
```diff
@@ -4,6 +4,7 @@ git:
 language: ruby
 bundler_args: --without development testext production --deployment --jobs=4 --retry=5
 cache: bundler
+dist: precise
```
Comment (davidlimcheng) [inline .travis.yml]: Hey @lyttam, @raydavis fixed the Travis issues on #6776, so I think we can remove this configuration for now.  Saves us a PR down the road.

Comment (lyttam) [inline .travis.yml]: Thanks @davidlimcheng, I removed it.

**Comment by davidlimcheng** [comment]
Nice @lyttam

**Pull request merged**